src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -134,10 +135,10 @@ def get_issue_table_contents(issue_list: List[Dict[str, int]]) -> List[PullReque
 # TODO(cathy): Change the client typing to allow for multiple SCM Integrations
 def safe_for_comment(
     gh_client: GitHubAppsClient, repository: Repository, pull_request: PullRequest
-) -> bool:
+) -> Tuple[bool, JSONData]:
```

Comment:
Are we returning JSON? Or a list of JSON?

Reply:
`JSONData` is actually a wrapper for `Any`... can update this in a follow up